src/functionalTest/resources/scenarios/RIA-6509-change-non-detained-aaa-to-detained-aaa-edit-appeal.json

```diff
@@ -0,0 +1,44 @@
+{
+  "description": "RIA-6509-change-non-detained-aaa-to-detained-aaa-edit-appeal",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentativeOrgSuccess",
+    "input": {
+      "eventId": "editAppeal",
+      "state": "appealStarted",
+      "caseData": {
+        "legalRepresentativeEmailAddress": "{TEST_LAW_FIRM_ORG_SUCCESS_USERNAME}",
+        "template": "minimal-age-assessment-appeal-started.json",
+        "replacements": {
+          "appellantInDetention": "Yes",
```

Comment:
this should be no as well

Reply:
This is the scenario where we started as non-detained appeal but in edit we mark it as detained. hence appellantInDetention = Yes in replacements.